### PR TITLE
Fix auto scale create with fix date without  recurrence

### PR DESCRIPTION
**Pull request by yuwzho**
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

azure_rm_autoscale

##### ANSIBLE VERSION

```paste below
2.7.0
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by zikalino** [comment]
a kind of difficult to understand what the bug was exactly

**Comment by yuwzho** [comment]
@zikalino When using this module to create an auto scale setting with a fixed date, the code construct the setting instance without recurrence, So here need to check whether the recurrence is `none`, if so, the code needs to set the recurrence in the autoscalesettings to None.

File: lib/ansible/modules/cloud/azure/azure_rm_autoscale.py [hunk 1]
```diff
@@ -550,8 +550,9 @@ def create_rule_instance(params):
                                                                schedule=(RecurrentSchedule(time_zone=p.get('recurrence_timezone'),
                                                                                            days=p.get('recurrence_days'),
                                                                                            hours=p.get('recurrence_hours'),
-                                                                                           minutes=p.get('recurrence_mins')))
-                                                               if p.get('recurrence_frequency') else None)) for p in self.profiles or []]
+                                                                                           minutes=p.get('recurrence_mins'))))
```
Comment (zikalino) [inline lib/ansible/modules/cloud/azure/azure_rm_autoscale.py:553]: so this parameter was in totally wrong place right?

**Review by zikalino — APPROVED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_autoscale.py (hunk 1 above)
Comment (yuwzho) [reply lib/ansible/modules/cloud/azure/azure_rm_autoscale.py:553]: Here add another conditional check for one case

**Pull request merged**